### ISSUE TITLE
Why does the loss remain unchanged during the DPO process?

### ISSUE THREAD

**Issue by shiqichen17** [Yifan-Song793/ETO#14]
I observed that during the DPO process, the loss consistently remains at 0.6931. The logs during this phase are as follows:

`{'loss': 0.6931, 'learning_rate': 5e-07, 'rewards/chosen': 0.0, 'rewards/rejected': 0.0, 'rewards/accuracies': 0.0, 'rewards/margins': 0.0, 'logps/rejected': 0.0, 'logps/chosen': 0.0, 'logits/rejected': -1.3516827821731567, 'logits/chosen': -1.3605669736862183, 'epoch': 0.59}`.

**Comment by hehaha68** [comment]
> I observed that during the DPO process, the loss consistently remains at 0.6931. The logs during this phase are as follows:
> 
> `{'loss': 0.6931, 'learning_rate': 5e-07, 'rewards/chosen': 0.0, 'rewards/rejected': 0.0, 'rewards/accuracies': 0.0, 'rewards/margins': 0.0, 'logps/rejected': 0.0, 'logps/chosen': 0.0, 'logits/rejected': -1.3516827821731567, 'logits/chosen': -1.3605669736862183, 'epoch': 0.59}`.

The same problem, do you solve it?
